src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/config/TestConfigurationIntegrationTest.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.config;
+
+public class TestConfigurationIntegrationTest {
```

Comment:
This could be named a lot better.  It's not really a Test it just contains some global constant values used within other tests.